docs/source/_static/style_admonitions.css

```
@@ -0,0 +1,34 @@
+:root {
+  --icon--shortcuts: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' height='16' fill='currentColor' class='bi bi-signpost' viewBox='0 0 16 16'%3E%3Cpath d='M7 1.414V4H2a1 1 0 0 0-1 1v4a1 1 0 0 0 1 1h5v6h2v-6h3.532a1 1 0 0 0 .768-.36l1.933-2.32a.5.5 0 0 0 0-.64L13.3 4.36a1 1 0 0 0-.768-.36H9V1.414a1 1 0 0 0-2 0zM12.532 5l1.666 2-1.666 2H2V5h10.532z'/%3E%3C/svg%3E");
+  --icon--returned-in: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' height='16' fill='currentColor' class='bi bi-arrow-return-right' viewBox='0 0 16 16'%3E%3Cpath fill-rule='evenodd' d='M1.5 1.5A.5.5 0 0 0 1 2v4.8a2.5 2.5 0 0 0 2.5 2.5h9.793l-3.347 3.346a.5.5 0 0 0 .708.708l4.2-4.2a.5.5 0 0 0 0-.708l-4-4a.5.5 0 0 0-.708.708L13.293 8.3H3.5A1.5 1.5 0 0 1 2 6.8V2a.5.5 0 0 0-.5-.5z'/%3E%3C/svg%3E");
+}
+.admonition.shortcuts {
+  border-color: rgb(43, 155, 70);
+}
+.admonition.shortcuts > .admonition-title {
+  background-color: rgba(43, 155, 70, 0.1);
+  border-color: rgb(43, 155, 70);
+}
+.admonition.shortcuts > .admonition-title::before {
+  background-color: rgb(43, 155, 70);
+  -webkit-mask-image: var(--icon--shortcuts);
+  mask-image: var(--icon--shortcuts);
+}
+
+.admonition.returned-in {
+  border-color: rgb(230, 109, 15);
+  max-height: 200px;
+  overflow-y: scroll;
+}
+.admonition.returned-in:hover {
+  cursor: move;
+}
+.admonition.returned-in > .admonition-title {
+  background-color: rgba(177, 108, 51, 0.1);
+  border-color: rgb(230, 109, 15);
```

```suggestion
  border-color: rgb(230, 109, 15);
  position: sticky;
  top: 0;
  backdrop-filter: blur(10px);
```

Another minor suggestion: That way the header doesn't scroll away. The text gets blurred when passing underneath it. Would like it even better if it would just vanish, but with the transparency of the title color and my limited CSS knowledge, I couldn't manage that xD

---

The blurring works in Chrome but doesn't work in Firefox on my Ubuntu. We could make the heading opaque, but then its colors must depend on whether the theme is light or dark

---

https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter#browser_compatibility

okay, firefox 103 was released in [2022-07](https://en.wikipedia.org/wiki/Firefox_version_history#Firefox_102_through_114), so fairly recent … All other browser support it since ~3 more years though … I guess in FF the blurring is just not applied and the text slides through the background unchanged?

---

I think I figured out another way :) See 7e30f91297a2bfc3a9bb6e1a9c17173510b794a8